dataflow/snippets/noxfile_config.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default TEST_CONFIG_OVERRIDE for python repos.
+
+# You can copy this file into your directory, then it will be imported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7", "3.7", "3.9", "3.10", "3.12"],
```

Comment:
suggestion: we recommend testing on oldest (3.8) and newest (3.12) supported version of Python. See https://devguide.python.org/versions/

Reply:
I'm testing against 3.11 because AFAIK Beam doesn't support 3.12 yet:

https://github.com/apache/beam/issues/29149

Specifically, with 3.12, I get build errors from distutils module not found, which I'm pretty sure is an issue in the Beam SDK compatibility.

Reply:
SG, thanks for the explanation.